### PR TITLE
Tickets

### PR DESCRIPTION
**Pull request by GigiDev7**

### CODE REVIEW

File: server/services/flightRecord.js [hunk 1]
```diff
@@ -11,7 +12,34 @@ const createFlightRecord = async (flightData) => {
...
+      standartTickets: flightData.standartTickets,
+      economTickets: flightData.economTickets,
     });
+    for (let i = 0; i < flightData.businessTickets; i++) {
+      const newTicket = {
+        price: flightData.businessPrice,
+        flightRecordId: newFlight._id,
+        ticketClass: "business",
+      };
+      await Ticket.create(newTicket);
```
Comment (Lokki17) [inline server/services/flightRecord.js]: Use insertMany instead of single adding

File: server/services/flightRecord.js [hunk 2]
```diff
@@ -11,7 +12,34 @@ const createFlightRecord = async (flightData) => {
...
+    }
+    for (let i = 0; i < flightData.standartTickets; i++) {
+      const newTicket = {
+        price: flightData.standartPrice,
+        flightRecordId: newFlight._id,
+        ticketClass: "standart",
+      };
+      await Ticket.create(newTicket);
+    }
+    for (let i = 0; i < flightData.economTickets; i++) {
```
Comment (Lokki17) [inline server/services/flightRecord.js]: could you move this 3 loops in one function/method?

**Review by Lokki17 — CHANGES_REQUESTED** [review]
Fix remarks

File: server/services/flightRecord.js (hunk 2 above)
Comment (GigiDev7) [reply server/services/flightRecord.js]: yes sure

File: server/services/flightRecord.js [hunk 3]
```diff
@@ -1,17 +1,83 @@
...
+  stTickets,
+  ecTickets,
+  bsPrice,
+  stPrice,
+  ecPrice,
+  id
+) {
+  const tickets = [];
+
+  for (let i = 0; i < bsTickets; i++) {
```
Comment (Lokki17) [inline server/services/flightRecord.js]: I dont like this approach. What if we'll need to add 4-th class? 5-th? You'll need to add new the same code here. We need code which won't depend on count of classes. Do you understand? Maybe it's better to use objects to discribe groups of classes. Something like { class: ..., countOfSeats: ... } (maybe extra field price). In that case it enougth to have one handler.

File: server/services/flightRecord.js [hunk 4]
```diff
@@ -1,17 +1,83 @@
...
   const airplane = await Airplane.findOne({ company: flightData.airline });
+  const {
+    flightId,
+    businessTickets,
+    standartTickets,
+    economTickets,
+    businessPrice,
+    standartPrice,
+    economPrice,
+  } = flightData;
```
Comment (Lokki17) [inline server/services/flightRecord.js]: The same remark. It shoud be universal code without linking with specific class

**Review by Lokki17 — CHANGES_REQUESTED** [review]
Think on remarks

File: server/services/flightRecord.js (hunk 3 above)
Comment (GigiDev7) [reply server/services/flightRecord.js]: fixed it

**Review by Lokki17 — APPROVED** [review]

**Pull request merged**